test/tests/kernels/2d_diffusion/tests

```diff
@@ -5,27 +5,48 @@
     exodiff = 'out.e'
     scale_refine = 5
     group = 'requirements'
+    requirement = 'MOOSE shall test Dirichlet BC for a 2D diffusion problem.'
+    issues = '#6397'
+    design = '/DirichletBC.md'
   [../]
 
   [./testneumann]
     type = 'Exodiff'
     input = '2d_diffusion_neumannbc_test.i'
     exodiff = 'neumannbc_out.e'
     scale_refine = 5
+    requirement = 'MOOSE shall test Neumann BC for a 2D diffusion problem.'
```

Comment:
MOOSE shall provide an ability to solve a 2D diffusion problem with Neumann boundary conditions.